src/master_gateway.js

```diff
@@ -266,6 +268,7 @@ var masterGateway = {
 
     slaveLoaded : function(event, slaveData) {
         if (!slaveData.loaded || this.slaves[slaveData.frameId].isLoaded) {
+            logger.out('warn', '[GG] Master:', 'Slave is already loaded!', event.data);
```

Comment:
This could be now false positive if `loaded` callback is not provided.

Reply:
@dikaso Okay, so shall I destructure this into if/else if block and add logger only in second case?

Reply:
Agree.